avalon/tools/creator/app.py

```diff
@@ -227,6 +227,13 @@ def _on_data_changed(self):
                 subset_name = subset_name[0].upper() + subset_name[1:]
             result.setText("{}{}".format(family, subset_name))
 
+            if not subset_name:
+                subset.setStyleSheet("border-color: #3A3939;")
```

Comment:
Is this the case of having no border-color whatsoever? If so, then wouldn't `subset.setStyleSheet("")` do the trick to reset any overrides?

Reply:
I did not see that ! Fixed in commit 4726466